fix(css): update `!important` vs layers note (#34987)
mdn/content: files/en-us/web/css/cascade/index.md

Now the paragraph will be blue. The `!important` in the earliest declared layer takes precedence of subsequent layers and unlayered important declarations. If the inline style contained !important, such as `<p style="color: black !important">`, again the paragraph would be black. Inline importance does take precedence over all other author declared `!important` declarations, no matter the specificity.

> > **Note:** `!important` reverses the precedence of cascade layers. For this reason, try not to use `!important` to override external styles. Instead, import the external style sheets(from frameworks, widget stylesheets, libraries, etc.) into initial layers, which will demote their precedence, and author-defined layers will have higher precedence. `!important` should only be used sparingly, if ever, to guard required styles against later overrides, in the first declared layer.

Reviewer: ```suggestion
> **Note:** The `!important` flag reverses the precedence of cascade layers. For this reason, try not to use `!important` to override external styles. Instead, import external style sheets (from frameworks, widget stylesheets, libraries, etc.) into named layers. Importing stylesheets into a layer as the first declaration in your CSS demotes their precedence, and author-defined layers, defined later in your CSS, will have higher precedence. The `!important` flag should only be used sparingly, if ever, to guard required styles against later overrides, in the first declared layer.
```

`initial` wasn't clear. My suggestion is wordy, but i think clarifies it?

Reviewer: created issue https://github.com/mdn/content/issues/35005

Reviewer: This edit will fix https://github.com/mdn/content/issues/16129

Presentational attributes are attributes in the source document that can affect styling. For example, when included, the deprecated `align` attribute sets the alignment on several HTML elements and the `fill` attribute defines the color used to paint SVG shapes and text and defines the final state for SVG animations. While they are author styles, presentational attributes do not participate in the cascade.

> If the HTML presentation attribute is supported by the user agent, valid presentational attributes included in HTML are translated to the corresponding CSS rules, such as the [`align`](/en-US/docs/Web/HTML/Element/img#align) or [`fill`](/en-US/docs/Web/SVG/Attribute/fill) property (all SVG presentation attributes can be used as a CSS property) inserted in the author stylesheet prior to any other styles with a specificity equal to `0`.

Reviewer: ```suggestion
If the HTML presentation attribute is supported by the user agent, valid presentational attributes included in HTML and SVG, such as the [`align`](/en-US/docs/Web/HTML/Element/img#align) or [`fill`](/en-US/docs/Web/SVG/Attribute/fill) attributes, are translated to the corresponding CSS rules (all SVG presentation attributes are supported as CSS properties) and inserted in the author stylesheet prior to any other styles with a specificity equal to `0`.
```

> If the HTML presentation attribute is supported by the user agent, valid presentational attributes included in HTML are translated to the corresponding CSS rules, such as the [`align`](/en-US/docs/Web/API/HTMLImageElement/align) or [`fill`](/en-US/docs/Web/SVG/Attribute/fill) property (all SVG presentation attributes can be used as a CSS property) inserted in the author stylesheet prior to any other styles with a specificity equal to `0`.

Reviewer: how about https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#align for align?
I plan on creating the fill css property this week. Not sure if it's premature to macro it? https://github.com/orgs/mdn/projects/25/views/1?pane=issue&itemId=71536919

Author: ```suggestion
If the HTML presentation attribute is supported by the user agent, valid presentational attributes included in HTML are translated to the corresponding CSS rules, such as the [`align`](/en-US/docs/Web/HTML/Element/img#align) or [`fill`](/en-US/docs/Web/SVG/Attribute/fill) property (all SVG presentation attributes can be used as a CSS property) inserted in the author stylesheet prior to any other styles with a specificity equal to `0`.
```
